src/shims/unix/macos/foreign_items.rs

```diff
@@ -180,11 +180,11 @@ pub trait EvalContextExt<'tcx>: crate::MiriInterpCxExt<'tcx> {
                 let res = if this.pthread_setname_np(
                     thread,
                     this.read_scalar(name)?,
-                    this.eval_libc("MAXTHREADNAMESIZE").to_target_usize(this)?.try_into().unwrap(),
+                    LibcError("MAXTHREADNAMESIZE").to_target_usize(this)?.try_into().unwrap(),
```

Comment:
`to_target_usize(this)` is not implemented for `IoError`

Reply:
Yeah leave these unchanged, they don't set errno.